src/Data/Text/Internal/Fusion/Common.hs

```diff
@@ -12,7 +12,25 @@
 -- API or name. Functions in this module may not check or enforce
 -- preconditions expected by public modules. Use at your own risk!
 --
--- Common stream fusion functionality for text.
+-- This module provides a common stream fusion interface for text.
+-- The stream interface allows us to write text pipelines which
+-- do no allocate intermediate text values. For example, we could
+-- guarantee no intermediate text is allocated by writing the following:
+--
+-- @
+--   getNucleotides :: `Data.Text.Internal.Text` -> `Data.Text.Internal.Text`
+--   getNucleotides =
+--         `Data.Text.Internal.Fusion.unstream`
+--       . `filter` isNucleotide
+--       . `toLower`
+--       . `Data.Text.Internal.Fusion.stream`
+--     where
+--       isNucleotide chr =
+--         chr == \'A\' || chr = \'a\' ||
```

Comment:
you use `toLower`, so `chr` cannot be `ACGT`
also there is a typo: `=` vs `==`

Reply:
Thanks very much, I'll fix it this evening.

Reply:
Hopefully now fixed.